zerver/actions/create_realm.py

```diff
@@ -242,6 +244,11 @@ def do_create_realm(
             realm=realm,
             event_type=RealmAuditLog.REALM_CREATED,
             event_time=realm.date_created,
+            extra_data={
+                "how_did_you_find_zulip": how_did_you_find_zulip,
+                # Only motivation to store this separately is to avoid displaying this on /activity page.
+                "how_did_you_find_zulip_other_description": how_did_you_find_zulip_other_description,
```

Comment:
Other text is not displayed on `/activity` right now since it would make the rows fall out of place.